### DISCUSSION TITLE
Set custom resolution

### DISCUSSION THREAD

**Discussion by willko22** [lllyasviel/Fooocus#1587] [Q&A] [answered]
Hello.
Is there a way to set custom resolutions? or higher resolutions?
if yes how?

**Comment by Irmagon** [comment, marked as answer]
> Hello. Is there a way to set custom resolutions? or higher resolutions? if yes how?

Two way
1. set custom resolution in config.txt i.e. 
```
"available_aspect_ratios": [
    "768*768",
    "768*1024",
    "768*1344"
    ],
```

2. Owerride resolution in Advanced/Advanced/Developer Debug Mode
Forced Overwrite of Generating Width
Forced Overwrite of Generating Height
(dnt affect in Extreme Speed)

**Reply by willko22** [reply]
Thank you.
And do you know what is the maximum i can use for the images to look normaly?

**Reply by Irmagon** [reply]
In fooocus by default sets **optimal resoution** for stable generation.
If u need more-and-more resolution - try step by step upscale in fooocus, or use any other app (_Topaz GigapixelAi almost best_)

**Reply by carlmoss22** [reply]
hi guys, i would like to add 1280/1280 to foocus but when i add this to the config text and override resoution in advanced debug mode nothing happens.

**Comment by SGThirkell** [comment]
> hi guys, i would like to add 1280/1280 to foocus but when i add this to the config text and override resoution in advanced debug mode nothing happens.

You only have to add to the config.txt and not override as well. Have you tried adding just the resolution on it's own?

**Reply by carlmoss22** [reply]
thank you for your answer. i add this in (like mentioned in the post above) in the (otherwise empty)config file but nothing show up in the ui.

 "available_aspect_ratios": [
    "1280*1280",
    "768*1024",
    "768*1344"
    ],

i do it now with the override. works also.
